Engine/EditorUI/Helper/Billboard.cpp

```diff
@@ -18,20 +18,20 @@ Billboard::~Billboard()
 
 void Billboard::Render(const float3& position) const
 {
-	GLuint shader_program = App->program->GetShaderProgramId("Billboard");
```

Comment:
Why is this commented? if it was because it was not working, it should be fix already

Reply:
it made the engine crash, we can test if is working now

Reply:
If i uncomment the code now doesnt crash but the billboards are not rendered :(